sdk/src/ordinals/indexer.ts

```diff
@@ -0,0 +1,324 @@
+/**
+ * Base path for Ordinals regtest Indexer.
+ * @default "http://0.0.0.0:80"
+ */
+export const REGTEST_ORD_BASE_PATH = "http://0.0.0.0:80";
+
+/**
+ * @ignore
+ */
+export interface InscriptionsData {
+    inscriptions: string[];
+    prev: string;
+    next: string;
+    lowest: number;
+    highest: number;
+}
+
+/**
+ * @ignore
+ */
+export interface InscriptionUTXO {
+    value: number;
+    script_pubkey: string,
+    address:string,
+    transaction:string,
+    sat_ranges: string,
+    inscriptions: string[];
```

Comment:
This is supposed to be inscription content, right? In that case I would suggest adding a new type for that, so it does not get confused with inscription ids, something like `type InscriptionContent = string;` which can be later used for `inscriptions: InscriptionContent[]`